cypress/integration/view-and-manage-evidence.spec.ts

```diff
@@ -57,8 +57,10 @@ describe('Can view and manage evidence', () => {
     cy.get('figure').should('contain', '24.7 KB');
     // cy.get('a').should('contain', 'Open in new tab').and('have.attr', 'href');
 
-    // TODO: uncomment when History is implemented
-    // cy.get('h2').should('contain', 'History');
+    cy.get('h2').should('contain', 'History');
+    cy.get('table')
+      .should('contain', 'Retention expires')
```

Comment:
Here we could also check for _Valid until_, we must have missed that test when we implemented DES-222

Reply:
Ok, I'll add that